v3/src/components/graph/adornments/movable-value/movable-value-adornment-registration.tsx

```diff
@@ -73,7 +73,7 @@ const Controls = () => {
 
 registerAdornmentContentInfo({
   type: kMovableValueType,
-  plots: ['dotPlot'],
+  plots: ['dotPlot', 'linePlot'],
```

Comment:
```suggestion
  plots: ['dotPlot'],
```
You shouldn't need this. `isCompatibleWithPlotType()` handles extending `dotPlot` to other compatible plot types like `linePlot` (there are actually four of them).